client/src/components/FormInput.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import EmailIcon from '../assets/email.svg';
+import PasswordIcon from '../assets/password.svg';
+import InputField from './InputField';
+
+const Container = styled.div`
+  width: 22%;
```

Comment:
might be annoying later if you would like to reuse this component somewhere else. `FormInput` suggests that this is a component that may be used in any form as any type of input. `width: 22%;` should be declared at a higher level to make this a more flexible component

Reply:
yeah I know the size props aren't finally decided here. I made the mistake to first create the single form components. Next step would be to create the form around it and let it decide the sizes...